lib/cfd_trading/cfd_order_confirmation.dart

```diff
@@ -66,24 +70,39 @@ class CfdOrderConfirmation extends StatelessWidget {
                         mainAxisAlignment: MainAxisAlignment.end,
                         children: [
                           ElevatedButton(
-                              onPressed: () {
+                              onPressed: () async {
                                 // switch index to cfd overview tab.
                                 cfdTradingService.selectedIndex = 1;
-                                // todo send order to maker.
+
+                                // TODO: Plug in actual state changes
 
                                 // clear draft order from cfd service state
                                 cfdTradingService.draftOrder = null;
 
+                                try {
+                                  // TODO: Don't hardcode the values
+                                  await api.openCfd(takerAmount: 20000, leverage: 2);
```

Comment:
Not sure if we should keep the hardcoded value or use the actual margin. I would vote for using the actual margin but keep the leverage locked in at 2 until we action #198 

I also captured https://github.com/itchysats/10101/issues/200 which is related.

@luckysori if we keep the hardcoded value please add a comment to https://github.com/itchysats/10101/issues/200 :)

Reply:
Let's keep it hard-coded until we action #200. I'll add the comment.